watcher.go

```diff
@@ -0,0 +1,107 @@
+package frankenphp
+
+import (
+	fswatch "github.com/dunglas/go-fswatch"
+	"go.uber.org/zap"
+	"sync/atomic"
+	"sync"
+)
+
+// latency of the watcher in milliseconds
+const watcherLatency = 150
+
+var (
+	watchSessions      []*fswatch.Session
+	blockReloading     atomic.Bool
+	reloadWaitGroup    sync.WaitGroup
+)
+
+func initWatcher(watchOpts []watchOpt, workerOpts []workerOpt) error {
+	if(len(watchOpts) == 0 || len(workerOpts) == 0) {
+		return nil
+	}
+
+	watchSessions := make([]*fswatch.Session, len(watchOpts))
+	for i, watchOpt := range watchOpts {
+		session, err := createSession(watchOpt, workerOpts)
+		if(err != nil) {
+			return err
+		}
+		watchSessions[i] = session
+	}
+
+	for _, session := range watchSessions {
+		go session.Start()
+	}
```

Comment:
```suggestion
	watchSessions := make([]*fswatch.Session, len(watchOpts))
	for i, watchOpt := range watchOpts {
		session, err := createSession(watchOpt, workerOpts)
		if(err != nil) {
			return err
		}
		watchSessions[i] = session

		go session.Start()
	}
```

Reply:
The original idea was to only start the sessions if there are no errors while creating them

Reply:
ok makes sens!